data/json/items/magazine/shot.json

```diff
@@ -160,5 +160,100 @@
     "ammo_type": [ "shot" ],
     "flags": [ "MAG_COMPACT" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 5 } } ]
+  },
+  {
+    "id": "mossberg_590m_mag_5",
+    "copy-from": "mossberg_590m_mag_10",
+    "type": "MAGAZINE",
+    "name": { "str": "magazine-fed pump shotgun 5-round magazine" },
```

Comment:
I would suggest that specifying “magazine-fed” in the name of the magazine is a little redundant.

Reply:
i plan to add more, omw to make `revolver magazine-fed shotgun` aka SRM Arms Model 1216 at least, adn differentiate between almost 3 mag fed shotguns (and potentially more) would be pretty important i think?